.asf.yaml

```diff
@@ -0,0 +1,36 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+notifications:
+  commits:      commits@cassandra.apache.org
+  issues:       commits@cassandra.apache.org
+  pullrequests: pr@cassandra.apache.org
+  jira_options: link worklog
+
+github:
+  description: "Sidecar for Apache Cassandra®"
+  homepage: https://cassandra.apache.org/
+  enabled_merge_buttons:
+    squash:  false
+    merge:   false
+    rebase:  true
+  features:
+    wiki: false
+    issues: false
+    projects: false
+  autolink_jira:
+    - CASSANDRASC
```

Comment:
maybe `CASSANDRA` too …? (i doubt CASSANDRA-nnnn references would mean anything else here…)

Reply:
When there are 2 projects listed, does it link to both projects? Given all tickets for sidecar are supposed to be created in `CASSANDRASC`.

Reply:
even if work is done under a `CASSANDRASC` ticket, other tickets may still be mentioned.

the following implies multiple project linking works…
https://github.com/apache/infrastructure-asfyaml?tab=readme-ov-file#autolinks-for-jira

Reply:
no, it just links to what you create a commit message for. I get that 99% it is CASSANDRASC but there _can_ be a ticket CASSANDRA-XYZ which _might_ touch sidecar as well (like CASSANDRA-19854, right?) so it will link that one. If we kept just CASSANDRASC then this one would not be linked.

Reply:
what Mick said :)

Reply:
Thanks for the example.